src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -219,8 +225,18 @@ public StreamingResponseBody streamEvents(
                 final EventConsumer eventConsumer = topicRepository.createEventConsumer(
                         kafkaQuotaClientId,
                         streamConfig.getCursors());
+
+                final String bytesFlushedMetricName = MetricRegistry.name(
```

Comment:
This name looks like "lola.stups_some-application-name.some#event-ype-name.bytes-flushed". The existent methods in `MetricUtils` looks like "nakadi.subscriptions.subscription-id.bytes-flushed". It has 2 problems:

1. it has no space for application-id
2. it has no space for lola/hila.

I could create new methods there, to have it all in a single class, though. Is this what you meant?